client/src/main/java/io/pravega/client/admin/impl/ReaderGroupManagerImpl.java

```diff
@@ -43,10 +44,10 @@
     private final ClientFactory clientFactory;
     private final Controller controller;
 
-    public ReaderGroupManagerImpl(String scope, URI controllerUri) {
+    public ReaderGroupManagerImpl(String scope, URI controllerUri, ConnectionFactory connectionFactory) {
```

Comment:
I think that if we remove the new parameter to the `ControllerImpl` constructor as suggested below, we will be able to get rid of these changes here.